docs/user_guide/dask.rst

```diff
@@ -0,0 +1,205 @@
+.. _dask:
+
+Lazy Computation with Dask
```

Comment:
Can we rename this section to "Usage with Dask"?  Although lazy computation is a result of using Dask, it isn't the main user goal.